app/assets/javascripts/flood_risk_engine/application.js

```diff
@@ -12,3 +12,11 @@
 //
 //= details.polyfill
 //= require_tree .
+
+
+$(function() {
```

Comment:
The flood risk engine state machine does that via the go_back event - and for the journey up until now has worked OK. The problem is that the manual address page is outside of the steps system.

There could also be an inconsistency if you went back with the link and then back with the browser button, as that would take you forward! 

Making both the back link and the back button do the same thing makes these problems much easier to handle, while if JavaScript is switched off the link still works, but with a couple of peculiarities.